endpoints-support.md

```diff
@@ -17,319 +17,319 @@
 <br /> 
  
 ## HPE OneView 
-| Endpoints                                                                                       |           Verb            |          V2600           |          V2800           |          V3000           |          V3200           |          V3400           |          V3600           |          V3800           |          v4000           | v4200| v4400|         v4600       |
-| ----------------------------------------------------------------------------------------------- | :-----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: |:----------------------:| :----------------------:|:----------------------:|
+| Endpoints                                                                                       |           Verb            |          V2600           |          V2800           |          V3000           |          V3200           |          V3400           |          V3600           |          V3800           |          v4000           |          v4200        |        v4400      |         v4600       |        v4800         |
+| ----------------------------------------------------------------------------------------------- | :-----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: | :----------------------: |:----------------------:| :----------------------:|:----------------------:|:----------------------:|
 | **Appliance Configuration Timeconfig**                                                          | 
-| <sub>/rest/appliance/configuration/timeconfig/locales</sub>                                     |            GET            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |:white_check_mark:| :white_check_mark:|:white_check_mark:|
```

Comment:
I can see some extra white check marks in endpoints-support file. Please check it.